packages/types/src/schemas/schemas.ts

```diff
@@ -1265,171 +1265,341 @@ export const MatrixConfigSchema = {
 		},
 		"presentationOptions": {
 			"description": "Allows for conditional styling",
-			"type": "object",
```

Comment:
any idea what's going on here. Is it just formatting?

Reply:
This is when I ran `generate` on the types package, because the build was failing because the types had changed